asyncpg/connection.py

```diff
@@ -1755,6 +1755,10 @@ async def connect(dsn=None, *,
                   max_cacheable_statement_size=1024 * 15,
                   command_timeout=None,
                   ssl=None,
+                  sslcert=None,
```

Comment:
There's no need for all the extra arguments in `connect()`.  If you need to customize the certs, you are supposed to pass an appropriate `SSLContext` in the `ssl` argument.